frontend/public/extend/devconsole/components/Pipelines/PipelineDetails.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import {
+  SectionHeading,
+  ResourceSummary
+} from '../../../../components/utils';
+
+//const TaskRow = ({obj: task}) =><><dd>{task.name}</dd><dt>Ref: {task.taskRef.name}</dt></>;
+
+const DevPipelineDetails = ({obj:pipeline}) => <div className="co-m-pane__body">
+<div className="row">
+  <div className="col-sm-6">
+    <SectionHeading text="Pipeline Overview" />
+    <ResourceSummary resource={pipeline}>
```

Comment:
If there are no children you don't need a separate end tag. 
```suggestion
    <ResourceSummary resource={pipeline} />
```